crypto/homo/paillier/paillier.go

```diff
@@ -41,6 +44,8 @@ var (
 	ErrInvalidInput = errors.New("invalid input")
 	//ErrInvalidMessage is returned if the message is invalid
 	ErrInvalidMessage = errors.New("invalid message")
+	//ErrSmallPublicKeySize is returned if the size of public key is small
+	ErrSmallPublicKeySize = errors.New("small the size of public key")
```

Comment:
errors.New("small the size of public key") ->  errors.New("small public key")?

Reply:
done